### PR TITLE
Make changes to brokerconfig func

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

Fixes #

## Proposed Changes

-
-
-

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

File: control-plane/pkg/reconciler/broker/broker.go [hunk 1]
```diff
@@ -518,7 +517,8 @@ func (r *Reconciler) topicConfig(logger *zap.Logger, broker *eventing.Broker) (*
 
 	topicConfig, err := kafka.TopicConfigFromConfigMap(logger, cm)
 	if err != nil {
-		if isRebuilt {
+		// Check if the rebuilt CM is empty
+		if cm !=nil && len(cm.Data) == 0 {
```
Comment (matzew) [inline control-plane/pkg/reconciler/broker/broker.go:521]: the `rebuilt` CM is always an object - but in errors is empty

**Review by pierDipi — COMMENTED** [review]
style / Auto-format and Check is red

LGTM

**Comment by matzew** [comment]
@pierDipi updated :see_no_evil:

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

**Pull request merged**